### ISSUE TITLE
Cannot open configure file in msisensor-pro 1.2.0

### ISSUE THREAD

**Issue by Dichromatism** [xjtu-omics/msisensor-pro#36]
Hi, 

I have tried msisensor-pro (both installed by conda and pulled from docker). When I was using hg19 as a reference and tried to make a baseline using my .bam files,

`docker run pengjia1110/msisensor-pro msisensor-pro baseline -d microsatellites_new.list -i normal_2.tsv -o baseline`

the software returned following errors:    

`Check for the environment ... 
Current work path: /
Microsatellites file: /microsatellites_new.list
Configure file path: /normal_2.tsv
Output path:/baseline
/baseline/ is not existing！ now make it! OK!
/baseline/detail is not existing！ now make it ! OK!

Load files ... 
Open configure file failed, please provide valid configure file ! `

I am sure that my .bam files were indexed correctly and the names were `*.sorted.dedup.bam` and `*.sorted.dedup.bam.bai`.

When I using version 1.0.2 it returned as cannot find the index files.

So what should I do to figure this out? Thanks!

**Comment by Dichromatism** [comment]
Thanks to your reply!

I ran msisensor-pro successfully using `pro` method. The baseline were constructed using 33 normal bams. I tested it using one of my paired data. When I using `msisensor msi` (is this one equal to `msisensor-pro msi`?) to check msi sites with my normal.bam and tumor.bam, the results returned as 76 out of 196 sites were MSI-H. However when I using  `msisensor-pro pro` with my tumor.bam and the baseline which I just constructed, the results returned as 5 out of 180 sites were MSI-H. Which of those results should I believe?

I noticed the `msisensor-pro pro`  had different defaults in `-m` and `-w` to `msisensor msi`, so I changed those values but the results were same as  before.

Hope to your further reply!
